src/KMonad/Prelude/Imports.hs

```diff
@@ -17,10 +17,6 @@ import RIO as X hiding
     view, ASetter, ASetter', Lens, Getting, Lens'
   , SimpleGetter, lens, over, set, sets, to, (^.)
 
-    -- The following line is required for newer stack releases.
-    -- This is also the reason for the OPTIONS_GHC pragma
-    -- , (^..), (^?), preview, (%~), (.~)
-
```

Comment:
I think straight up removing this is not the way to go, because then people who want to compile kmonad on GHC 8.10 (possibly 8.8 as well, haven't tested) will have to add this themselves according to the error messages.  Perhaps we need a tiny bit of `CPP` here?

Reply:
Oops! I thought I was on a newer version of GHC and I warn't getting any errors. Turns out I was still on the old one. I've added this back in, with the GHC-pragma, which I prefer over CPP for now.

I'm really hoping to restrict all use of CPP to only modules named `OS`. This might be folly, but I'd like to keep trying for now.